Changelog.md

```diff
@@ -1,5 +1,19 @@
 ## Ignition Gazebo 6.x
 
+### Gazebo Sim 6.12.0 (2022-08-30)
+
+1. Add topic parameter to thrust plugin
+    * [Pull request #1681](https://github.com/gazebosim/gz-gazebo/pull/1681)
+
+1. Add information about `<topic>` system paramter
+    * [Pull request #1671](https://github.com/gazebosim/gz-gazebo/pull/1671)
+
+1. Adding tests for hydrodynamics
```

Comment:
I think this could also mention that the plugin's behavior has changed, and we can add an entry to the migration guide.